### PR TITLE
Update getting-started.md

### PR DESCRIPTION
**Pull request by alyssoncm**
-Removing migration from hosted Parse information 
-Adding some information to better explain  Parse Server to new users

### CODE REVIEW

File: _includes/parse-server/getting-started.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
 # Getting Started
 
-Parse Server is an open source version of the Parse backend that can be deployed to any infrastructure that can run Node.js. You can find the source on the [GitHub repo](https://github.com/parse-community/parse-server).
+Parse Server is an open source backend that can be deployed to any infrastructure that can run Node.js. You can find the source on the [GitHub repo](https://github.com/parse-community/parse-server).
 
-* Parse Server is not dependent on the hosted Parse backend.
-* Parse Server uses MongoDB directly, and is not dependent on the Parse hosted database.
-* You can migrate an existing app to your own infrastructure.
+* Parse Server uses MongoDB/PostgreSQL as a database.
```
Comment (mtrezza) [inline _includes/parse-server/getting-started.md]: I think this can be reworded to "MongoDB or PostgreSQL".

**Review by mtrezza — CHANGES_REQUESTED** [review]
Thanks for the PR. Just a minor change.

File: _includes/parse-server/getting-started.md (hunk 1 above)
Comment (alyssoncm) [reply _includes/parse-server/getting-started.md]: ok

**Review by mtrezza — APPROVED** [review]
Thanks for this PR! Let's see whether @TomWFox gives his blessings.

**Review by TomWFox — APPROVED** [review]
That's been annoying me for sooo long!

**Pull request merged**